src/verinfast/config.py

```diff
@@ -289,7 +289,6 @@ def init_argparse(self) -> argparse.ArgumentParser:
             "-t", "--truncate", "--truncate_findings",
             dest="truncate_findings",
             type=int,
-            default=-1,
```

Comment:
This is important to remove! Args trump Config. If we set a default value for any arg it will override the config.